### ISSUE TITLE
Prevent multiple parallel --renew runs on same domain

### ISSUE THREAD

**Issue by defnull** [acmesh-official/acme.sh#4898]
Running `acme.sh --renew` multiple times in parallel may break the `$domain.conf` file and prevent future runs from succeeding. We faced this issue after a manual and a cron-triggered run happened at the same time, mangling the `$domain.conf` file. It would be nice if `acme.sh` could prevent this from happening, e.g. by detecting parallel runs and exiting early with an error if a conflict is detected. Perhaps some per-domain lock as explained here: https://www.putorius.net/lock-files-bash-scripts.html

**Comment by defnull** [comment]
This affects all versions and debug output does not provide any additional info.

**Comment by neilpang** [comment]
Closing as stale: no activity for over 18 months. If this still applies on the current release, please open a new issue with the `--debug 2` log.
